perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -3929,6 +3933,7 @@ public boolean removeAttributeWithoutCheck(PerunSession sess, Host host, Attribu
 			} catch (WrongAttributeValueException | WrongReferenceAttributeValueException ex) {
 				throw new InternalErrorException(ex);
 			}
+			log.info("{} removed this attribute {} from  host {}", sess.getLogId(), attribute.getName(), host.getId());
```

Comment:
I would remove unnecessary "this" from the message (and other similar messages). Also there is double space before "host" word.